package.json

```diff
@@ -22,60 +22,74 @@
     "@alrra/travis-scripts": "3.0.1",
     "@google-cloud/storage": "1.6.0",
     "ajv": "5",
-    "babel-core": "6.25.0",
-    "babel-eslint": "8.1.2",
-    "babel-loader": "7.1.1",
+    "babel-core": "6.26.3",
+    "babel-eslint": "8.2.3",
+    "babel-loader": "7.1.4",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "babel-preset-env": "1.6.0",
+    "babel-preset-env": "1.6.1",
     "babel-preset-flow": "6.23.0",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-2": "6.24.1",
-    "chalk": "2.3.0",
+    "chalk": "2.4.1",
     "chromeless": "1.4.0",
     "convict": "4.0.1",
     "coveralls": "2.13.1",
-    "css-loader": "^0.28.4",
-    "eslint": "4.14.0",
+    "css-loader": "0.28.11",
+    "eslint": "4.19.1",
```

Comment:
should only be a peer dep

Reply:
leaving as `dependency` only. not in `peerDep`

Reply:
actually I guess eslint specifically should be a devDependency.

Reply:
another issue with making eslint a peerDependency is that it's installed as a devDependency in mup-web and devDependencies seemingly don't satisfy peer dependencies.